src/components/WwwFrame/PromotionalBanner/Banners/DepositIncentiveBanner.vue

```diff
@@ -27,15 +40,24 @@ export default {
 		return {
 			amountToLend: 0,
 			isLoggedin: false,
+			basketTotal: 0,
 		};
 	},
 	inject: ['apollo', 'cookieStore'],
 	apollo: {
 		query: amountToLendQuery,
 		preFetch: true,
+		variables() {
+			return {
+				basketId: this.cookieStore.get('kvbskt')
+			};
+		},
 		result({ data }) {
 			this.amountToLend = data?.my?.depositIncentiveAmountToLend ?? 0;
 			this.isLoggedin = !!data?.my?.id ?? false;
+			this.basketTotal = data.shop?.basket?.items?.values?.reduce((sum, item) => {
```

Comment:
This should use the loan reservation total like you do below. However, that total is a string, so it will need to be parsed into a number first:
```
this.basketTotal = parseFloat(data.shop?.basket?.totals?.loanReservationTotal ?? 0);
```